apps/.eslintignore

```diff
@@ -5,3 +5,5 @@
 /lib
 /build
 node_modules
+
+!.storybook
```

Comment:
Was curious why this needed to be added and was going to ask about it, but then saw the explanation [here](https://github.com/storybookjs/eslint-plugin-storybook/blob/a053ba9ec53fa782467fdf25c58231ecf24b5277/README.md?plain=1#L58). 😄